website/templates/base.mako

```diff
@@ -139,14 +139,23 @@
      ## TODO: shouldn't always have the watermark class
     ${self.content_wrap()}
 
+<div id="IEDepreciationBanner"  class="alert warningBanner">
+    <div class="warningBannerText">
+        The OSF no longer supports Internet Explorer please switch to another browser or continue at your own risk.
+    </div>
+    <div class="warningBannerAcceptBtn">
+        <div class="btn btn-default" data-dismiss="alert" data-bind="click: accept" aria-label="Accept">Accept</div>
+    </div>
+</div>
```

Comment:
I saw this text on Chrome when I was logged out, on `/register/`.
Chrome:
<img width="1275" alt="screen shot 2019-01-09 at 11 56 19 am" src="https://user-images.githubusercontent.com/9755598/50918599-4ca8c900-1406-11e9-8575-b03eac7af85a.png">

Reply:
I can't reproduce this, do you have any additional details? what's in the console and exact url. are you in Incognito Mode?

Reply:
Not incognito, I had just logged out and then clicked on "sign up". URL was http://localhost:5000/register/.  Let me know if you still can't reproduce and i'll switch back to your branch

Reply:
@pattisdr Sorry to bother you, but I still can't reproduce, it could be possible css changes that allow stacking will prevent this from happening incidentally.

Reply:
yes, I think it's fixed!